packages/cli/lib/prompts/projectDevTargetAccountPrompt.js

```diff
@@ -12,133 +12,145 @@ const { logger } = require('@hubspot/local-dev-lib/logger');
       });
     }
-
-    choices = [
-      ...devTestAccounts.map(mapNestedAccount),
-      {
-        name: i18n(`${i18nKey}.createNewDeveloperTestAccountOption`),
-        value: {
-          targetAccountId: null,
-          createNewDeveloperTestAccount: true,
-        },
-        disabled: disabledMessage,
+  }
+  // Order choices by Developer Sandbox -> Standard Sandbox
+  choices = [
+    ...sandboxAccounts
+      .filter(a => a.accountType === HUBSPOT_ACCOUNT_TYPES.DEVELOPMENT_SANDBOX)
+      .map(mapNestedAccount),
+    ...sandboxAccounts
+      .filter(a => a.accountType === HUBSPOT_ACCOUNT_TYPES.STANDARD_SANDBOX)
+      .map(mapNestedAccount),
+    {
+      name: i18n(`${i18nKey}.createNewSandboxOption`),
+      value: {
+        targetAccountId: null,
+        createNestedAccount: true,
       },
-    ];
-  } else {
-    let sandboxUsage = {};
-    try {
-      sandboxUsage = await getSandboxUsageLimits(defaultAccountId);
-    } catch (err) {
-      logger.debug('Unable to fetch sandbox usage limits: ', err);
-    }
+      disabled: disabledMessage,
+    },
+    {
+      name: i18n(`${i18nKey}.chooseDefaultAccountOption`),
+      value: {
+        targetAccountId: defaultAccountId,
+        createNestedAccount: false,
+      },
+    },
+  ];
 
-    const sandboxAccounts = accounts
-      .reverse()
-      .filter(
-        config =>
-          isSandbox(config) && config.parentAccountId === defaultAccountId
-      );
-    let disabledMessage = false;
+  return selectTargetAccountPrompt(
+    defaultAccountId,
+    'sandbox account',
+    choices
+  );
+};
 
-    if (
-      sandboxUsage['DEVELOPER'] &&
-      sandboxUsage['DEVELOPER'].available === 0
-    ) {
-      if (sandboxAccounts.length < sandboxUsage['DEVELOPER'].limit) {
-        disabledMessage = i18n(`${i18nKey}.sandboxLimitWithSuggestion`, {
-          authCommand: uiCommandReference('hs auth'),
-          limit: sandboxUsage['DEVELOPER'].limit,
-        });
-      } else {
-        disabledMessage = i18n(`${i18nKey}.sandboxLimit`, {
-          limit: sandboxUsage['DEVELOPER'].limit,
-        });
-      }
-    }
-    // Order choices by Developer Sandbox -> Standard Sandbox
-    choices = [
-      ...sandboxAccounts
-        .filter(
-          a => a.accountType === HUBSPOT_ACCOUNT_TYPES.DEVELOPMENT_SANDBOX
-        )
-        .map(mapNestedAccount),
-      ...sandboxAccounts
-        .filter(a => a.accountType === HUBSPOT_ACCOUNT_TYPES.STANDARD_SANDBOX)
-        .map(mapNestedAccount),
-      {
-        name: i18n(`${i18nKey}.createNewSandboxOption`),
-        value: {
-          targetAccountId: null,
-          createNewSandbox: true,
-        },
-        disabled: disabledMessage,
-      },
-      {
-        name: i18n(`${i18nKey}.chooseDefaultAccountOption`),
-        value: {
-          targetAccountId: defaultAccountId,
-          createNewSandbox: false,
-        },
-      },
-    ];
+const selectDeveloperTestTargetAccountPrompt = async (
```

Comment:
Found the old setup a little confusing so I separated the prompts for selecting sandboxes and selecting test accounts

Reply:
yeah definitely good call 👍